### DISCUSSION TITLE
Modelling porous regions using HVAC?

### DISCUSSION THREAD

**Discussion by SaiSantosh1824** [firemodels/fds#12166] [General]
Hi All,

I have a wall with a louvered section, having a louver efficiency of 0.4 (i.e., allowing 40% of flow through it). I have been modelling this as an opening with the computed free area. However, is there a way to model this using HVAC elements?

Thank you!

**Comment by drjfloyd** [comment]
Define a duct for the flow path and assign it losses such that you get the correct pressure drop for the louver.
